### PR TITLE
flake8 3.7.5

### PR DESCRIPTION
**Pull request by lembacon**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
Removed entrypoints<0.4.0,>=0.3.0 from https://files.pythonhosted.org/packages/b4/ef/063484f1f9ba3081e920ec9972c96664e2edb9fdc3d8669b0e3b8fc0ad7c/entrypoints-0.3.tar.gz#sha256=c70dd71abe5a8c85e55e12c19bd91ccfeec11a6e99044204511f9ed547d48451 (from flake8==3.7.5) from build tracker '/private/tmp/pip-req-tracker-8pndan6u'
Removed build tracker '/private/tmp/pip-req-tracker-8pndan6u'
Command "/usr/local/Cellar/flake8/3.7.5/libexec/bin/python3.7 /usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip install --ignore-installed --no-user --prefix /private/tmp/pip-build-env-1cndy4zf/overlay --no-warn-script-location -v --no-binary :all: --only-binary :none: -i https://pypi.org/simple -- flit" failed with error code 1 in None
Exception information:
Traceback (most recent call last):
  File "/usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip/_internal/cli/base_command.py", line 176, in main
    status = self.run(options, args)
  File "/usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip/_internal/commands/install.py", line 315, in run
    resolver.resolve(requirement_set)
  File "/usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip/_internal/resolve.py", line 131, in resolve
    self._resolve_one(requirement_set, req)
  File "/usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip/_internal/resolve.py", line 294, in _resolve_one
    abstract_dist = self._get_abstract_dist_for(req_to_install)
  File "/usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip/_internal/resolve.py", line 242, in _get_abstract_dist_for
    self.require_hashes
  File "/usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip/_internal/operations/prepare.py", line 349, in prepare_linked_requirement
    abstract_dist.prep_for_dist(finder, self.build_isolation)
  File "/usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip/_internal/operations/prepare.py", line 125, in prep_for_dist
    "Installing build dependencies"
  File "/usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip/_internal/build_env.py", line 195, in install_requirements
    call_subprocess(args, show_stdout=False, spinner=spinner)
  File "/usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip/_internal/utils/misc.py", line 761, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip._internal.exceptions.InstallationError: Command "/usr/local/Cellar/flake8/3.7.5/libexec/bin/python3.7 /usr/local/Cellar/flake8/3.7.5/libexec/lib/python3.7/site-packages/pip install --ignore-installed --no-user --prefix /private/tmp/pip-build-env-1cndy4zf/overlay --no-warn-script-location -v --no-binary :all: --only-binary :none: -i https://pypi.org/simple -- flit" failed with error code 1 in None
```

**Comment by lembacon** [comment]
@BrewTestBot test this please

File: Formula/flake8.rb [hunk 1]
```diff
@@ -16,11 +16,35 @@ class Flake8 < Formula
...
+    sha256 "5e8c00e30c464c99e0b501dc160b13a14af7f27d4dffb529c556e30a159e231d"
+  end
+
   def install
     venv = virtualenv_create(libexec, "python3")
-    system libexec/"bin/pip", "install", "-v", "--no-binary", ":all:",
-                              "--ignore-installed", buildpath
-    system libexec/"bin/pip", "uninstall", "-y", name
+    resource("entrypoints").stage do
+      rm_f "pyproject.toml"
```
Comment (lembacon) [inline Formula/flake8.rb]: Without removing this file, `pip` will ignore the `setup.py` file and attempt to download the [`flit`](https://github.com/takluyver/flit) build system.

There is a `--no-use-pep517` option for `pip`, however that will yield:

```
18:25:24 Disabling PEP 517 processing is invalid: project specifies a build backend of setuptools.build_meta in pyproject.toml
```

**Comment by lembacon** [comment]
@fxcoudert Mind taking a look at this? Thanks!

File: Formula/flake8.rb (hunk 1 above)
Comment (fxcoudert) [reply Formula/flake8.rb]: This looks OK, but a short version of that comment should be in the formula itself :)

**Review by fxcoudert — APPROVED** [review]

File: Formula/flake8.rb (hunk 1 above)
Comment (javian) [reply Formula/flake8.rb]: @lembacon could you add this ?

**Comment by lembacon** [comment]
Added. 😅

**Comment by lembacon** [comment]
💚 Merging.

**Pull request closed without merging**